### DISCUSSION TITLE
Analoge Funktion zu WebGIS4 "about.aspx" in WebGIS5?

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#79] [Q&A]
Gibt es eine zu "about.aspx" analoge Funktion in WebGIS5?
Die Antwort dieser Funktion hat sich bestens in Monitoring-Tools einbauen lassen.
(z.B. kann man auf den Text "Application running since" testen).

**Comment by gis-eni** [comment]
zb:

/api/rest bzw /api/rest?f=json

**Comment by faulgurke** [comment]
Gibt's eine andere schlaue Möglichkeit WebGIS5 zu monitoren (d.h. frühzeitig auf Probleme aufmerksam zu werden)?

**Comment by gis-eni** [comment]
Ich weiß nicht was du monitoren willst? Ob die Applikation läuft und ob Dienste Fehler haben etc. bekommt man aus dem oben stehenden Link.

**Reply by faulgurke** [reply]
Die Frage zielt auf best practise Lösungen oder Erfahrungen von anderen Nutzern ab.
